app/controllers/api/v2/users_controller.rb

```diff
@@ -119,6 +119,21 @@ def destroy
         end
       end
 
+      api :DELETE, "/users/:id/invalidate_jwts", N_("Invalidates user's JSON web tokens")
```

Comment:
perhaps it should be PUT, it's kind of an update of a user, we're not really deleting any tokens, not sure how users feel about that though, the permission we're checking here is also edit_users...